src/main/java/user/controller/UserController.java

```diff
@@ -13,34 +14,35 @@
 @Controller("/user")
 public class UserController {
 
-    @Inject UserRepository userRepository;
+    @Inject
+    UserService userService;
 
     @Get("/{id}")
     public HttpResponse getUserbyId(int id){
-        return HttpResponse.ok().body(userRepository.findById(id));
+        return HttpResponse.ok().body(userService.findById(id));
     }
 
     @Get("/")
-    public Iterable<User> index() {
-        return userRepository.findAll();
+    public Iterable<UserDTO> index() {
+        return userService.showAllUser();
     }
 
     @Post("/addUser")
     public HttpResponse addUser(@Body @Valid User user){
-        userRepository.save(user);
+        userService.add(user);
         return HttpResponse.ok();
     }
 
     @Delete("/{id}")
     public HttpResponse deleteUserbyId(@NotNull int id){
-        userRepository.deleteById(id);
+        userService.deleteById(id);
         return HttpResponse.ok();
     }
 
     @Put("/{id}")
     public HttpResponse editUser(@Body User user, int id){
         user.setId(id);
-        userRepository.update(user);
+        userService.editUser(user);
         return HttpResponse.ok();
```

Comment:
Instead of returning just HttpResponse.ok(), you can send the update data in HttpResponse body.

Reply:
Resolved.